src/foundation_models/CROMA/use_croma.py

```diff
@@ -20,9 +20,9 @@ def __init__(
         """
         super().__init__()
         # check types
-        assert isinstance(pretrained_path, str), (
-            f"pretrained_path must be a string, not {type(pretrained_path)}"
-        )
+        # assert isinstance(pretrained_path, str), (
```

Comment:
Could also use random CROMA model and not just pretrained.

Reply:
Also probably could remove the entire logic, and just use the torchgeo CROMA model then we only need an import and can remove all this code

Reply:
Actually, does not work yet, because in torchgeo the list of output features is not supported, which is needed for segmentation.